packages/dart_frog_cli/test/src/commands/dev/dev_test.dart

```diff
@@ -158,14 +157,15 @@ void main() {
     });
 
     test('fails if hostname is invalid', () async {
-      when(() => runner.start()).thenAnswer((_) => Future.value());
+      when(() => runner.start(any())).thenAnswer((_) => Future.value());
       when(() => runner.exitCode).thenAnswer(
         (_) => Future.value(ExitCode.success),
       );
 
       when(() => argResults['hostname']).thenReturn('ticarica');
       when(() => argResults['port']).thenReturn('1234');
       when(() => argResults['dart-vm-service-port']).thenReturn('5678');
+      when(() => argResults.rest).thenReturn(['--enable-experiment=macros']);
```

Comment:
Nit: Could we just provide nothing for this test?

```suggestion
      when(() => argResults.rest).thenReturn(['']);
```